drivers/age-swift/Tests/age-swiftTests/age_swiftTests.swift

```diff
@@ -0,0 +1,24 @@
+import XCTest
+@testable import age_swift
+
+let host = "127.0.0.1";
+let port = 5432 ;
+let user = "root";
+let db_name = "demo";
+
+final class age_swiftTests: XCTestCase {
+    func testExample() throws {
+        // This is an example of a functional test case.
+        // Use XCTAssert and related functions to verify your tests produce the correct
+        // results.
+        let age = Age()
+
+        let connection_param:[String:Any] = ["host":host,"port":port, "user":user, "dbname":db_name]
```

Comment:
How about having a type to represent the connection params? 

So that the IDE (Xcode) can code-complete these? Rather than the user having to remember the name of strings, and perhaps type them incorrectly?

Reply:
Makes sense. Will try to do this too in my upcoming pull request